### ISSUE TITLE
Add support for [plot](https://www.npmjs.com/package/plot

### ISSUE THREAD

**Issue by DonJayamanne** [DonJayamanne/typescript-notebook#23]
https://www.npmjs.com/package/plot

**Comment by DonJayamanne** [comment]
@randomfractals is this the package you were suggesting?

**Comment by RandomFractals** [comment]
no, I meant @mbostock's new Plot lib: https://observablehq.com/@observablehq/plot

See this repo: https://github.com/observablehq/plot

I have a set of demos you can check out on ObservableHQ: https://github.com/RandomFractals/Chicago-transportation-notebooks

Data summary notebook renderer I ported last month uses it: https://github.com/RandomFractals/vscode-data-table#data-summary--renderer-view
